src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/dto/nomis/NomisReport.kt

```diff
@@ -34,6 +35,19 @@ data class NomisReport(
   @Schema(description = "The date and time the incident was reported")
   val reportedDateTime: LocalDateTime,
 
+  @Schema(description = "The date and time the incident was created")
+  val createDateTime: LocalDateTime,
+  @Schema(description = "The username of the person who created the incident")
+  val createdBy: String,
+
+  @Schema(description = "The date and time the incident was last updated")
+  val lastModifiedDateTime: LocalDateTime? = createDateTime,
+  @Schema(description = "The username of the person who last updated the incident")
+  val lastModifiedBy: String? = createdBy,
+
+  @Schema(description = "The follow up date for the incident")
+  val followUpDate: LocalDate? = null,
```

Comment:
should we leave a to-do message noting that we currently ignore this?

Reply:
Not sure, this is not the only field in the NOMIS sync request we are ignoring...it may get a bit too noisy if we add a note on each fields 🤔